src/redux/sagas/saga.js

```diff
@@ -32,10 +32,10 @@ function* watchAddAlbum(){
 
 
 
-function* fetchImages(ID) {
+function* fetchImages(albumID) {
     let albumImages = [];
       try {
-        const response = yield fetch('https://api.imgur.com/3/album/'+ID, {
+        const response = yield fetch('https://api.imgur.com/3/album/'+albumID, {
         method: 'GET',  
         headers: {
           'Authorization': 'Client-ID 77248e7a549713b'},
```

Comment:
Try to use the Javascript function to iterate an array like (forEach, filter, ...)

Reply:
Error will return empty array now. An empty array will trigger a default message on screen.